### DISCUSSION TITLE
Mouse cursor is black ,how fix it? xrdp-0.9.15 with  ./configure --enable-neutrinordp

### DISCUSSION THREAD

**Discussion by lockon563** [neutrinolabs/xrdp#2685] [Q&A]
xrdp-0.9.15 with  ./configure --enable-neutrinordp  Proxy Module
Mouse cursor is black, how to  fix?  鼠标变成黑底了
![image](https://github.com/neutrinolabs/xrdp/assets/44391421/a763d7d7-5964-4994-b092-0dfa7afbdcb4)

![image](https://github.com/neutrinolabs/xrdp/assets/44391421/be903ff3-7fe2-4f7b-8fa7-118bc0f10429)

**Comment by matt335672** [comment]
Hi @lockon563 

This has been mentioned before - see #716, #1003, #1333. The last of these contains a client-side workaround in the first post:-

>I used the latest xrdp to connect windows 2012 server. When I enabled pointer shadow, the cursor turn to a black box. When I disabled pointer shadow, the cursor turn normal.

>Open Mouse Properties in Control Panel, click "Enable pointer shadow" and apply. The cursor immidiately turns a black box. If untick the "Enable pointer shadow", the cursor will turn normal.
